Pipfile

```diff
@@ -21,7 +21,7 @@ markdown = "==3.4.4"
 marshmallow = "==3.20.1"
 networkx = "==2.6"
 openpyxl = "==3.1.2"
-pandas = ">=2.0.2,<2.3.0"
+pandas = ">=1.3.5,<2.3.0"
```

Comment:
I thought the convention was to use Pipfile as the min version supported by Taipy for each dependency. So that means we should not have any range here, just the min version.
That is the role of the setup.py to define a range.